packages/suite-desktop/src-electron/libs/modules.ts

```diff
@@ -0,0 +1,43 @@
+import isDev from 'electron-is-dev';
+import { MODULES } from '@lib/constants';
+
+const modules = async (dependencies: Dependencies) => {
+    const { logger } = global;
+
+    logger.info('modules', `Loading ${MODULES.length} modules`);
+
+    await Promise.all(
+        MODULES.flatMap(async module => {
+            if (module.isDev !== undefined && module.isDev !== isDev) {
+                logger.debug(
+                    'modules',
+                    `${module.name} was skipped because it is configured for a diferent environment`,
+                );
+                return [];
+            }
+
+            const deps: { [name in keyof Dependencies]: any } = {};
+            module.dependencies.forEach((dep: keyof Dependencies) => {
+                if (dependencies[dep] === undefined) {
+                    logger.error(
+                        'modules',
+                        `Dependency ${dep} is missing for module ${module.name}`,
+                    );
+                    throw new Error(); // TODO
```

Comment:
TODO :)

Reply:
same question as in BaseProcess, is this error handled somewhere?

Reply:
Addressed in  417a0f7.